src/signals/incident-management/components/FilterForm/__tests__/FilterForm.test.js

```diff
@@ -40,6 +41,14 @@ jest.mock('models/categories/selectors', () => {
   }
 })
 
+const mockMAX_FILTER_LENGTHGetter = jest.fn()
```

Comment:
- [ ] consider calling this MAX_FILTER_LENGTHMock or something, easier to find. Also defining that's something is a getter is more like oop